carsons/carsons.py

```diff
@@ -311,6 +311,92 @@ def GMR_cn(self, phase) -> float:
         return (GMR_s * k * R**(k-1))**(1/k)
 
 
+class TapeShieldedCableCarsonsEquations(ModifiedCarsonsEquations):
+
+    ρ_tape_shield = 1.7721e-8  # copper resistivity at 20 degrees, ohm-meter
+
+    def __init__(self, model, *args, **kwargs):
+        super().__init__(model)
+        self.ds = model.tape_shield_outer_diameter
+        self.thickness = model.tape_shield_thickness
+
+        self.tape_shield_phases = model.tape_shield_outer_diameter.keys()
```

Comment:
i noticed you are assigning these but then the only thing they are used for is to pass into methods you call in the constructor. i wonder if we should access them inside those methods? or maybe we don't need them to become class attributes, just pass them from `model` into the methods